### PR TITLE
doc(readme): install and usage

### PR DESCRIPTION
**Pull request by aborsu**

### CODE REVIEW

**Comment by aborsu** [comment]
[?(syntax): Sagastandard](https://trello.com/c/vTOwdoQh/963-syntax-sagastandard)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,141 @@
...
+  lib/torrent.js:950:11: Expected '===' and instead saw '=='.
+```
+
+### Editor plugins
+
+There are currently no editor plugins directly supporting Saga-Sandard, but the following plugins can be customized to use it.
+
+- **Sublime users**: Try [SublimeLinter-contrib-semistandard](https://github.com/Flet/SublimeLinter-contrib-semistandard) for linting in your editor!
+- **Atom users** - Install [linter-js-standard](https://atom.io/packages/linter-js-standard)
+
```
Comment (satblip) [inline README.md:43]: can't we add something like : `ln -s /usr/local/bin/sagastandard /usr/local/bin/semistandard` to this section of the documentation? I know for sure that Sublime plugin is based on CLI calls. I don't know for Atom

File: README.md [hunk 2]
```diff
@@ -1,2 +1,141 @@
...
+- **NO console.log**
+- **NO debugger;***
+- **semicolons** check [Flet/semistandard]
+- Check [feross/standard] for the rest of the rules.
+
+## Usage
+
+The easiest way to use JavaScript Saga-Standard Style to check your code is to install it globally as a Node command line program. To do so, simply run the following command in your terminal (flag `-g` installs `sagastandard` globally on your system, omit it if you want to install in the current working directory):
+
+```bash
```
Comment (simondebaecke) [inline README.md:25]: Are lines 25 to 27 a desired duplicate from lines 6 to 9 ?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,141 @@
...
+
+```json
+{
+  "sagastandard": {
+    "parser": "babel-eslint"
+  }
+}
+```
+
+### [Vim](http://www.vim.org/)
```
Comment (simondebaecke) [inline README.md:81]: wow.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,141 @@
...
+set autoread
+```
+
+[vim-1]: https://github.com/scrooloose/syntastic
+
+### Ignoring files
+
+Just like in `standard`, The paths `node_modules/**`, `*.min.js`, `bundle.js`, `coverage/**`, hidden files/folders (beginning with `.`), and all patterns in a project's root `.gitignore` file are automatically excluded when looking for `.js` files to check.
+
+Sometimes you need to ignore additional folders or specific minfied files. To do that, add a `sagastandard.ignore` property to `package.json`:
```
Comment (simondebaecke) [inline README.md:103]: typo : minified

**Review by simondebaecke — CHANGES_REQUESTED** [review]

**Comment by simondebaecke** [comment]
Great doc ! 👍

File: README.md (hunk 2 above)
Comment (aborsu) [reply README.md:25]: For me its nice to have a quick install command at the top for people who don't want to read and a lengthier one for people who are less versed in the npm ecosystem.

Comment (simondebaecke) [reply README.md:25]: That's what I thought then, gg !

File: README.md [hunk 5]
```diff
@@ -1,2 +1,141 @@
-# sagastandard
-One Standard to bring them all, and in the darkness bind them.
+# Javascript Saga-Standard style
+### One Standard to bring them all, and in the darkness bind them.
+
+Based on [feross/standard] and [Flet/semistandard] with a few additionnal opiniated style rules.
```
Comment (simondebaecke) [inline README.md:4]: typo: opinionated

**Review by fbeuvens — APPROVED** [review]

**Pull request closed without merging**